assets/xml/scenes/misc/hiral_demo.xml

```diff
@@ -4,6 +4,16 @@
         <Scene Name="hiral_demo_scene_unused1" Offset="0x3580"/>
         <Scene Name="hiral_demo_scene_unused2" Offset="0x3640"/>
         <Scene Name="hiral_demo_scene_unused3" Offset="0x3700"/>
+
+        <Cutscene Name="gCutsceneMapKokiriEmeraldPart2Cs" Offset="0x0E50"/> <!-- Ganondorf riding the horse -->
+        <Cutscene Name="gCutsceneMapKokiriEmeraldPart3Cs" Offset="0x0100"/> <!-- World creation part 1 -->
```

Comment:
Personally, I sort of feel that the world creation cutscene should be considered it's own sequence, despite being part of the Kokiri Emerald sequence.

Reply:
I dont have too strong of an opinion here, but I think it adds a bit more complexity than is needed. its much easier to just label all the parts as the same continuous cutscene

Reply:
it makes sense but I think I still prefer naming it after where you come from normally when you watch it, also I'm not sure how to name them, like "world creation part 1" would be weird for the Ganondorf part, and having a completely different name could be confusing idk (I guess we can just add a comment about it)

Reply:
> it makes sense but I think I still prefer naming it after where you come from normally when you watch it, also I'm not sure how to name them, like "world creation part 1" would be weird for the Ganondorf part, and having a completely different name could be confusing idk (I guess we can just add a comment about it)

I wouldn't consider the introduction of Ganondorf to be part of the world creation/introduction of the triforce sequence. The creation sequence starts after, and is distinct in subject matter (it discusses the origin of the triforce, sacred realm) and that player input is disabled completely so that the sequences can be timed to the music.

But anyway, the short answer is that I guess `gCutsceneMapKokiriEmeraldPart3Cs` works if we just want to start getting things named, but I'm not particularly happy about it and would end up changing it in a PR later down the road.

My expectation is that these these longform cutscenes end up getting named with a little more detail and granularity than `CutscenePartN`. My thought was that for the longer chained sequences it'd be cleaner to treat them as sets of separate cutscenes. For example, the series of cutscenes that play after beating Ganon you'd perhaps split them into these: Seal Ganon, Credits Part 1, 2, 3, 4. But I guess that idea doesn't really work well with this cutscene at all, because after the creation cutscene you return back to the original cutscene. 

So now my ideal naming pattern for this cutscene would be something like this:

* `gCsAfterDekuTree_1_StartStory`
* `gCsAfterDekuTree_2_Ganondorf`
* `gCsAfterDekuTree_3_WorldCreation_1_desc`
* `gCsAfterDekuTree_3_WorldCreation_2_desc` and so on...
* `gCsAfterDekuTree_4_KokiriEmeraldAndDeath`

But then there's also the possibility that we might wanna encode the actor cue channel stuff into the cutscene name.

Reply:
yeah for now I'd like to, at least, have every cutscenes named instead of having things like `scene_cutsceneData_001234` in scenes

it's totally fine if the names I'm proposing are gone someday, for now it's just a matter of being able to identify something easily